### PR TITLE
fix/ deep-dark theme name issue 

### PR DESCRIPTION
**Pull request by MildTomato**
## I have read the [CONTRIBUTING.md](https://github.com/supabase/supabase/blob/master/CONTRIBUTING.md) file.

YES/NO

## What kind of change does this PR introduce?

Bug fix, feature, docs update, ...

## What is the current behavior?

Please link any relevant issues here.

## What is the new behavior?

Feel free to include screenshots if it includes visual changes.

## Additional context

Add any other context or screenshots.

### CODE REVIEW

File: packages/common/Providers.tsx [hunk 1]
```diff
@@ -7,7 +7,7 @@ export function ThemeProvider({ children, ...props }: ThemeProviderProps) {
   // @ts-ignore next-themes is old :/
 
   return (
-    <NextThemesProvider {...props} themes={['dark', 'light', 'deep-dark']}>
+    <NextThemesProvider {...props} themes={['dark', 'light']}>
```
Comment (ivasilov) [inline packages/common/Providers.tsx:10]: the `themes` prop overrides any themes passed in via props. Should the unstructured props be at the end?

**Review by ivasilov — CHANGES_REQUESTED** [review]
This PR doesn't solve the issue if the user has `deep-dark` in the localstorage value.

File: packages/common/Providers.tsx (hunk 1 above)
Comment (MildTomato) [reply packages/common/Providers.tsx:10]: we don't use 'classic-dark' on anything but Studio, so we override it in studio

File: packages/common/Providers.tsx [hunk 2]
```diff
@@ -7,7 +7,12 @@ export function ThemeProvider({ children, ...props }: ThemeProviderProps) {
   // @ts-ignore next-themes is old :/
 
   return (
-    <NextThemesProvider {...props} themes={['dark', 'light', 'deep-dark']}>
+    <NextThemesProvider
+      {...props}
+      themes={['dark', 'light']}
+      defaultTheme="dark"
+      storageKey={'supabase-theme'}
```
Comment (MildTomato) [inline packages/common/Providers.tsx:14]: @ivasilov this should fix it

Comment (MildTomato) [reply packages/common/Providers.tsx:14]: > This PR doesn't solve the issue if the user has deep-dark in the localstorage value.

what about now ?

**Review by ivasilov — APPROVED** [review]

File: packages/ui/build/css/themes/dark-combined.css [hunk 3]
```diff
@@ -1,3 +1,5 @@
+[data-theme='deep-dark'],
+.deep-dark,
```
Comment (MildTomato) [inline packages/ui/build/css/themes/dark-combined.css:2]: @ivasilov workaround without losing the saved themes for those unaffected

**Review by ivasilov — APPROVED** [review]

**Pull request merged**